packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -156,6 +164,8 @@ export class DocumentDeltaConnection
             });
 
         this.on("newListener", (event, listener) => {
+            assert(!this.closed, "can't add listeners after close");
```

Comment:
Event registration is a public interface, but the closed property is protected so it's impossible for a caller to know if their event registration will throw.  It's also atypical for event registration to throw (at least I can't think of any other precedent), so I'd prefer not to assert here.

Reply:
Better option would be to silently not register I think - it's a bit of a weird one because of the event forwarding thing.

Reply:
I like not-registering. Changing.